src/languages/r.js

```diff
@@ -178,8 +178,8 @@ export default function(hljs) {
               2: 'number'
             },
             match: [
-              PUNCTUATION,
-              NUMBER_TYPES
+              PUNCTUATION_RE,
+              NUMBER_TYPES_RE
```

Comment:
You'l have to actually rename them everywhere though :)

Reply:
On it, on it!